src/app/project/project-documents/project-document-resolver.services.ts

```diff
@@ -26,7 +26,7 @@ export class DocumentsResolver implements Resolve<Observable<object>> {
         pageNum,
         pageSize,
         sortBy,
-        { documentSource: 'PROJECT' },
```

Comment:
Just wondering why this was comply removed.

Reply:
Do you mean "completely?" This object specifies parts of the underlying mongoose model to filter by. By passing an empty object instead of this, we don't want to filter by any documentSource but get all documents(which are synonymous in this case, with files)